pkg/server/proto/sqlflow.proto

```diff
@@ -10,27 +10,29 @@ service SQLFlow {
     // SQL statements like `SELECT ...`, `DESCRIBE ...` returns a rowset.
     // The rowset might be big. In such cases, Query returns a stream
     // of RunResponse
+    // 
+    // SQLFlow implements the Run interface with two mode:
     //
+    // 1. Local model
```

Comment:
will the "local mode" move to couler "docker mode"?

Reply:
The `local mode` here is the current implementation. Will move to `docker mode` if the `docker mode` is ready in Couler.